dev-util/SPIRV-Headers/SPIRV-Headers-1.3.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils vcs-snapshot
```

Comment:
Pleas don't use vcs-snapshot for GitHub, S is predictable here.

Reply:
fixed